File: jsptest/src/main/java/controller/write.java
package controller;

import java.io.IOException;
import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

import model.dao.BoardDao;

@WebServlet("/board/write")
public class write extends HttpServlet {
	private static final long serialVersionUID = 1L;
       
    public write() {
        super();
    }

	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		//0. 인코딩
		request.setCharacterEncoding("UTF-8");
		
		//1. 요청
		String btitle = request.getParameter("btitle");
		String bcontent = request.getParameter("bcontent");
		String bwriter = request.getParameter("bwriter");
		String bpassword = request.getParameter("bpassword");
		
		//2. DAO
		BoardDao dao = new BoardDao();
		boolean result = dao.write(btitle, bcontent, bwriter, bpassword);
		
		//3. 응답
		response.getWriter().print(result);
	}

	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
		doGet(request, response);
	}

}

File: jsptest/src/main/webapp/js/write.js

function bwrite(){
	let btitle = document.querySelector('.btitle').value
	let bcontent = document.querySelector('.bcontent').value
	let bwriter = document.querySelector('.bwriter').value
	let bpassword = document.querySelector('.bpassword').value
	
	$.ajax({
		url : '/jsptest/board/write',
		data : {"btitle":btitle, "bcontent":bcontent, "bwriter":bwriter, "bpassword":bpassword},
		success : function(re) {
			if(re === 'true'){
				alert('글 작성 성공')
			}else{ alert('글 작성 실패') }
		}
		
	})
	
	
}



